test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test functionality on the cluster with different values of the --smp parameter on the nodes.
+"""
+import logging
+import time
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables
+from test.pylib.util import unique_name
+from test.topology.util import wait_for_token_ring_and_group0_consistency
+import pytest
+
```

Comment:
```
+from pytest import FixtureRequest
```

Reply:
Import needed for custom `RandomTables`.

Reply:
I don't understand your suggestion here, I import entire pytest - you suggest to avoid that?

Reply:
For the `request` fixture, I took it from another test and my suggestion below was without `pytest.`. No longer relevant.